README.md

```diff
@@ -19,49 +19,98 @@ You can find your API keys at <https://dashboard.razorpay.com/#/app/keys>.
 
 ```py
 import razorpay
-razor = razorpay.Client(auth=("<YOUR_API_KEY>", "<YOUR_API_SECRET>"))
+client = razorpay.Client(auth=("<YOUR_API_KEY>", "<YOUR_API_SECRET>"))
 ```
 
 
 ### Payments
 
-- Capture a payment
+- Fetch all payments
 
     ```py
-    razor.payment.capture("<PAYMENT_ID>", "<AMOUNT>")
+    client.payment.all()
     ```
 
 - Fetch a particular payment
 
     ```py
-    razor.payment.fetch("<PAYMENT_ID>")
+    client.payment.fetch("<PAYMENT_ID>")
     ```
 
-- Fetch all payments
+- Capture a payment
 
     ```py
-    razor.payment.all()
+    client.payment.capture("<PAYMENT_ID>", "<AMOUNT>")
+    Note: <AMOUNT> should be same as the original amount while creating the payment
     ```
 
-### Refunds
-
-- Initiate a refund
+- Refund a payment
 
     ```py
-    razor.refund.create("<PAYMENT_ID>")  # for whole amount
-    razor.refund.create("<PAYMENT_ID>", data={"amount": "<AMOUNT_TO_BE_REFUNDED>"})  # for particular amount
+    client.payment.refund("<PAYMENT_ID>", "<AMOUNT>") 
+    # for full refund
+
+    client.payment.refund("<PAYMENT_ID>", "<AMOUNT_TO_BE_REFUNDED>") 
+    # for particular amount
+
+    Note: <AMOUNT_TO_BE_REFUNDED> should be equal/less than the original amount
     ```
 
 - Fetch a particular refund
 
     ```py
-    razor.refund.fetch("<PAYMENT_ID>", "<REFUND_ID>")
+    client.payment.refund("<PAYMENT_ID>", "<REFUND_ID>")
```

Comment:
This is a little confusing. We are using the same refund call to both create a refund as well as to obtain a refund. This might be ok when we are designing rest apis, but might not be so in client api sdk. Why not use the fetch, as before instead of refund?